clone-replace.php

```diff
@@ -143,4 +145,62 @@ function cr_the_status( $post ) {
 		}
 	}
 
+	function cr_maybe_enqueue_gutenberg_script() {
+		wp_enqueue_script(
+			'clone-replace',
+			plugins_url( '/clone-replace/build/cloneReplace.js', __DIR__ ),
+			[ 'wp-blocks', 'wp-i18n' ],
+			'1.0.0',
+			true
+		);
+		inline_locale_data( 'clone-replace' );
+	}
+
+	add_action( 'enqueue_block_editor_assets', 'cr_maybe_enqueue_gutenberg_script', 10 );
```

Comment:
If the above is changed, then:
```suggestion
	add_action( 'enqueue_block_editor_assets', 'cr_action_enqueue_block_editor_assets', 10 );
```